packages/runfiles/index.js

```diff
@@ -0,0 +1,6 @@
+const runfileHelper = require(process.env['BAZEL_NODE_RUNFILES_HELPER']);
```

Comment:
this is an interesting choice. @gregmagolan had the same idea - but then you have some strange version skew coupling. If you upgrade `@bazel/runfiles` package by one version, you still get the code from whatever build_bazel_rules_nodejs package you're running. I suspect it's better if the sources move over here to packages/runfiles and users get the versioned thing. then internally we have a way to continue using it from our builtin code

Reply:
That makes sense. I'll update the PR.

Reply:
is this actually done? still seems like the runfiles implementation is coming out of the builtin instead of the npm package

Reply:
I was thinking if someone uses `@bazel/runfiles` it will be the helpers specific to the installed version of `@bazel/runfiles`. But if someone uses the environment variable (which we should just deprecate in the future IMO), they will get the version that is built-in.

Reply:
The actual runfile implementation is now in `packages/runfiles`